src/features/device-logs/lib/store.ts

```diff
@@ -193,6 +193,11 @@ function handleStreamingWrite(
 		try {
 			// Don't flush if the backend is reporting as unavailable
 			if (buffer.length && backend.available) {
+				const limit = ctx.retention_limit;
```

Comment:
Why is this referenced into a const here?

Reply:
I use it twice so it was just for having a shorter reference